modules/nw-using-ingress-forwarded.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ingress-controller
+
+[id="nw-using-ingress-forwarded_{context}"]
+= Using X-Forwarded headers
+
+The HAProxy Ingress Controller allows you to specify a policy for how to handle HTTP headers including `Forwarded` and `X-Forwarded-For`. The Ingress Operator uses `HTTPHeaders` field to configure the `ROUTER_SET_FORWARDED_HEADERS` environment variable of the Ingress Controller.
+
+.Procedure
+
+. Configure the `HTTPHeaders` field.
+.. Use the following command to edit the `IngressController` resource:
++
+[source,terminal]
+----
+$ oc edit IngressController
+----
++
+.. Under `spec`, set the `HTTPHeaders` policy field to `Append`, `Replace`, `IfNone`, or `Never`:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  name: default
+  namespace: openshift-ingress-operator
+spec:
+  httpHeaders:
+    forwarded:
+      policy: append
+----
+
+
+[discrete]
+== Example use cases
+
+*As a cluster administrator, you can:*
+
+* Configure an external proxy that injects the `X-Forwarded-For` header into each request before forwarding it to an Ingress Controller.
++
+To configure the Ingress Controller to pass the header through unmodified, You can specify the `never` policy. The Ingress Controller then never sets the headers, and applications receive only the headers that the external proxy provides.
+
+* Configure the Ingress Controller to pass the `X-Forwarded-For` header that your external proxy sets on extra-cluster requests through unmodified.
```

Comment:
Can you use a simpler term or phrase than extra-cluster? That term isn't used elsewhere in the collection.